themes/Atomize-color-theme.json

```diff
@@ -455,7 +455,12 @@
         "support.type.sys-types",
         "entity.other.inherited-class",
         "entity.name.type.class",
-        "entity.name.type"
+        "entity.name.type",
+        "entity.name.namespace",
+        "entity.name.scope-resolution.parameter.cpp",
+        "entity.name.scope-resolution.cpp",
+        "entity.name.scope-resolution.function.call.cpp",
+        "entity.name.scope-resolution.template.call.cpp"
```

Comment:
Sorry for the confusion here: I meant to move _only_ `entity.name.namespace` to this global list since it's a global modifier. These c++ specific ones can go back where you previously had them inside a C++ specific name.

Reply:
Oh, got it :)